packages/alfa-predicate-trilean/src/predicate-trilean.ts

```diff
@@ -0,0 +1,106 @@
+import { Equatable } from "@siteimprove/alfa-equatable";
+import { Mapper } from "@siteimprove/alfa-mapper";
+
+export type Trilean = boolean | undefined;
+
+export type PredicateTrilean<T, A extends Array<unknown> = Array<unknown>> = (
+  value: T,
+  ...args: A
+) => Trilean;
```

Comment:
This is already within a package named "trilean" so there's no need to repeat that in the name of the type. I'd simply make two modules within this package: `trilean.ts`, which defines the `Trilean` type, and `predicate.ts`, which defines the `Predicate` type. If consumers need to disambiguate between boolean predicates and trilean predicates, they can easily do so via import aliases.